src/main/java/org/smartregister/domain/PlanDefinition.java

```diff
@@ -48,6 +48,9 @@
 	@JsonProperty
 	@SerializedName("action")
 	private List<Action> actions;
+
+	@JsonProperty
+	private Boolean experimental;
```

Comment:
this should be a primitive and maybe false by default

Reply:
Changed to primitive and by default it is false.